test/codeu/chat/TestRunner.java

```diff
@@ -28,11 +28,21 @@ public static void main(String[] args) {
              codeu.chat.server.RawControllerTest.class,
              codeu.chat.util.TimeTest.class,
              codeu.chat.util.UuidTest.class,
-             codeu.chat.util.store.StoreTest.class
+             codeu.chat.util.store.StoreTest.class,
```

Comment:
When I try running TestRunner, it says the class can not be found or loaded. Not exactly sure what's causing this.